src/ontology/genepio-edit.owl

```diff
@@ -9840,11 +9843,12 @@ SubClassOf(obo:GENEPIO_0001843 ObjectSomeValuesFrom(obo:IAO_0000136 ObjectInters
 # Class: obo:GENEPIO_0001844 (origin)
 
 AnnotationAssertion(obo:IAO_0000112 obo:GENEPIO_0001844 "(0,0) is the origin of the cartesian coordinate system."@en)
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0001844 "\"In mathematics, the origin of a Euclidean space is a special point, usually denoted by the letter O, used as a fixed point of reference for the geometry of the surrounding space.
-
-In physical problems, the choice of origin is often arbitrary, meaning any choice of origin will ultimately give the same answer. This allows one to pick an origin point that makes the mathematics as simple as possible, often by taking advantage of some kind of geometric symmetry.\""@en)
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0001844 "A point in 0D space in which the origin is a fixed point of reference for the geometry of the surrounding space."@en)
```

Comment:
"A point in **zero-dimensional (0D)** space..."

Reply:
Change made :)

Reply:
Ok!  This was an ancient experimental term at beginning of GenEpiO when the possibility of having geospatial semantics directly in there as a feature of epidemiological investigations.